assets/projects.json

```diff
@@ -34,6 +35,21 @@
             "tag2": "C++ implementation of the Jupyter Kernel protocol",
             "tagDel": "C++ implementation of the Jupyter Kernel protocol"
         },
+
+
+	"xtensor-python": {
+        "type": "0",
+        "url": "https://github.com/QuantStack/xtensor-python",
+        "name": "Xtensor-python",
+        "md": "https://raw.githubusercontent.com/QuantStack/xtensor-python/master/README.md",
+        "documentation": "https://xtensor-python.readthedocs.io/en/latest/",
```

Comment:
Could your please change the link by removing the `/en/latest/` suffix for this project as well as all other projects listed on the page?